clickhouse-core/src/main/scala/com/clickhouse/io/ObservableOutputStream.scala

```diff
@@ -12,10 +12,9 @@
  * limitations under the License.
  */
 
-package xenon.clickhouse.io
-
-import xenon.clickhouse.Utils
+package com.clickhouse.io
 
+import com.clickhouse.Utils
```

Comment:
will this conflict with ClickHouse Java client? Do we need a sub-package?

Reply:
Good point 
Thanks